charter.md

```diff
@@ -100,10 +86,10 @@ Technically the Coq platform will be a git/github/gitlab repo containing:
 
 ## Steps
 
-- Move the current Windows Installer from core Coq to the platform
-- Simplify the windows installer in core Coq to only install Coq (and possibly very few selected plugins)
-- Ensure that platform CI runs with Coq master and release branches daily
-- Create a binary package for Linux (Snap store)
-- Create a binary package for Mac (Apple store)
-- Create an opam repo for the platform (or do this based on the existing Coq repo)
-- Provide developer setups on all platforms
+- [x] Move the current Windows Installer from core Coq to the platform
+- ~~Simplify the windows installer in core Coq to only install Coq (and possibly very few selected plugins)~~
```

Comment:
What does the double tilde mean here?

Maybe my wording was unclear. What I wanted to say is "decouple core Coq CI from all the packages the legacy Windows installer did build". This is implemented.

Reply:
The double tilde is strike-through. Indeed, this was not clear. I'll change the bullet point to your clearer statement and mark it as done.